src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -125,4 +132,24 @@ public Book updateBook(@RequestBody Book book, @PathVariable Long id) {
         findOne(id);
         return repository.save(book);
     }
+
+    /**
+     * Find book by isbn
+     *
+     * @param isbn: Book isbn (String)
+     *
+     * @return the book by the isbn
+     * @exception ResponseStatusException if a book don´t exists in api library
+     */
+    @GetMapping("/isbn/{isbn}")
+    public Book findByIsbn(@PathVariable String isbn, HttpServletResponse response) {
+        response.setStatus(200);
```

Comment:
I recommend using constants instead (check this out! https://docs.oracle.com/javaee/6/api/javax/servlet/http/HttpServletResponse.html) 😄